### PR TITLE
refactor(autoware_launch): add euclidean_clustering params

### PR DESCRIPTION
**Pull request by ismetatabay**
## Description

This PR adds the euclidean_cluster config files to autoware_launch package for migrating parameters from autoware_launch.
Autoware.Universe PR: https://github.com/autowarefoundation/autoware.universe/pull/4262
Releated: [#4250](https://github.com/autowarefoundation/autoware.universe/issues/4250)
## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by yukkysaito** [comment]
@ismetatabay Thank you for your PR. It is very nice. :+1:

File: autoware_launch/launch/components/tier4_perception_component.launch.xml [hunk 1]
```diff
@@ -10,23 +10,46 @@
...
+      name="object_recognition_detection_fusion_sync_param_path"
+      value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/detection/image_projection_based_fusion/roi_sync.param.yaml"
     />
-    <arg name="object_recognition_detection_fusion_sync_param_path" value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/detection/roi_sync.param.yaml"/>
+    <!-- tracking -->
     <arg
       name="object_recognition_tracking_multi_object_tracker_data_association_matrix_param_path"
       value="$(find-pkg-share autoware_launch)/config/perception/object_recognition/tracking/multi_object_tracker/data_association_matrix.param.yaml"
     />
+    <!-- segmentation -->
```
[the remark this answers is not in the document]
Comment (ismetatabay) [reply autoware_launch/launch/components/tier4_perception_component.launch.xml]: @yukkysaito Thanks a lot for your valuable suggestion, you are right :+1: . I divided comments and paths these 4 sub componenets at my latest commit.I will create another PRs for migrating parameters along autoware_launch, then i will group parameter paths with 4 sub-component topics. What dou you think?

[the remark this answers is not in the document]
Comment (yukkysaito) [reply autoware_launch/launch/components/tier4_perception_component.launch.xml]: Nice :+1: LGTM

**Review by yukkysaito — APPROVED** [review]
LGTM

**Pull request merged**